extension/src/popup/views/AccountHistory/index.tsx

```diff
@@ -166,7 +101,9 @@ export const AccountHistory = () => {
     fetchAccountHistory();
   }, [publicKey, networkDetails]);
 
-  return (
+  return isDetailViewShowing ? (
+    <TransactionDetail {...detailViewProps} />
```

Comment:
This is where we're controlling show/hide of TransactionDetail and pass the props that we constructed in `HistoryItem`